bundles/org.eclipse.e4.ui.model.workbench/src/org/eclipse/e4/ui/model/application/ui/basic/impl/PartImpl.java

```diff
@@ -1382,4 +1383,41 @@ public String toString() {
 		return result.toString();
 	}
 
-} //PartImpl
+	/**
+	 * <!-- begin-user-doc -->
+	 * If the method below changes, please ensure that {@link #toString()} is updated.
+	 * <!-- end-user-doc -->
+	 * @generated
+	 */
+	@Override
+	public String toStringGen() {
```

Comment:
This ensures that what's normally generated for toString() is generated here instead so that if the model changes, hopefully someone pays enough attention to read the documentation and to update the toString method above.